script.js

```diff
@@ -49,4 +49,80 @@ const photos = [
     },
 ];
 
-console.log('hello');
\ No newline at end of file
+console.log('hello');
+
+const gallery = document.getElementById("gallery");
+
+window.onload = loadImages;
+
+var loadMore_button = document.getElementById("loadMore")
+loadMore_button.addEventListener("click", loadImages)
+let button_clicks = 1;
+
+var all_button = document.getElementById("all");
+all_button.addEventListener("click", loadAllImages);
+
+var nature_button = document.getElementById("nature");
+nature_button.addEventListener("click", loadNature)
+
+var city_button = document.getElementById("city");
+city_button.addEventListener("click", loadCity)
+
+var animal_button = document.getElementById("animals");
+animal_button.addEventListener("click", loadAnimals)
+
+function loadImages(){
+    clearGallery();
```

Comment:
I tested your page and all the filters are working great! One thing I noticed, when the "Load more" button is clicked after one of the filters is clicked, it adds more photos that are not related to the filter. You should look into fixing that. Also, a nice-to-have feature would be to hide the "Load more" button if there are no more photos to show.